contrib/debian/rules

```diff
@@ -64,6 +64,9 @@ override_dh_install:
 	[ ! -d debian/tmp/usr/lib/modules-load.d ] || dh_install -pfwupd usr/lib/modules-load.d
 	[ ! -d debian/tmp/lib/modules-load.d ] || dh_install -pfwupd lib/modules-load.d
 	dh_missing -a --fail-missing
+	#enable GRUB instead of shim
+	[ ! -f debian/tmp/etc/fwupd/uefi_capsule.conf ] || sed -i 's,^#EnableGrubChainLoad=.*,EnableGrubChainLoad=true,' \
```

Comment:
I guess the question I have is whether we want this behavior turned on by default anywhere?  I feel like we shouldn't pick and choose packaged distros, it should be all or none.

Reply:
My gut instinct is "none" and have it easily enable-able/testable for users having problems with shim. Maybe change the default when it's been tested a bit.

Reply:
OK.  So eventually I think we also need to worry about the priority paths with COD, NVRAM, and GRUB.  Steady state my gut tells me we should have the order be to try these:
COD, GRUB, NVRAM

Reply:
I think CoD is the least well tested at the moment;lets release 1.6.2 defaulting to NVRAM and ask people to test chainloading in the release notes and report back.

Reply:
Yeah I'm just saying eventually that is where we want to be.  Another release or two.

Reply:
I'll drop the Debian bits that change the default and rebase it for you then,